SCHEDULING.md

```diff
@@ -14,9 +14,9 @@ Year | Month | Date | Venue* | Capacity | Event type
 2019 | March | 16th | [Elysian Coffee at 7th and Ontario](https://goo.gl/maps/vFcNsTtmu2C2) | 10 | Getting Started in Open Source 👩🏽‍💻
 2019 | April | [13th](https://www.meetup.com/nodeschool-vancouver/events/258713785/) | [Clio at Canada Way, Burnaby](https://goo.gl/maps/fCoVMK4GA7w) | 40 |
 2019 | April | 20th | [Elysian Coffee at 7th and Ontario](https://goo.gl/maps/vFcNsTtmu2C2) | 10 | Getting Started in Open Source 👩🏽‍💻
-2019 | May | 11th | To be announced | - |
+2019 | May | [11th](https://www.meetup.com/nodeschool-vancouver/events/260332010/) | [Sauce Labs at Water St and Abbott](https://osm.org/go/WJQrN1jlA?way=136258919) | 20 |
```

Comment:
Yay Open Street Maps

Reply:
I think you are praising yourself, i just copy and pasted the previous line